aksel.nav.no/website/pages/eksempler/bodylong/align.tsx

```diff
@@ -1,6 +1,14 @@
 import { BodyLong, VStack } from "@navikt/ds-react";
 import { withDsExample } from "components/website-modules/examples/withDsExample";
 
+function Descriptor({ children }) {
```

Comment:
Ikke så alt for viktig, men i eksemplene så bør `Example` fortsatt ligge først slik at de som ser på koden på Aksel.nav.no ser eksempelkoden først

Reply:
yeah... det slo meg halveis igjennom refactor... noen har mye grums som kommer forran det viktigste